<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormConfig</class>
 <widget class="QWidget" name="FormConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Setting</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>431</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="le_image_save_path"/>
     </item>
     <item>
      <widget class="QPushButton" name="pb_set_image_save_path">
       <property name="text">
        <string>select</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>set images capture save path:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>331</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>set threshold:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sp_threshold">
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_set_threshold">
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,1">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>set number of cameras:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sp_num_cam">
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_save_num_cam">
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_exit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
